src/main/java/com/google/sps/model/GmailClientImpl.java

```diff
@@ -83,6 +83,96 @@ public Message getUserMessage(String messageId, MessageFormat format) throws IOE
         .collect(Collectors.toList());
   }
 
+  // @Override
```

Comment:
Is this code supposed to be unused? If so, remove

Reply:
`Resolved`